recipes/genpy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "genpy" %}
+{% set version = "2016.1.3" %}
+{% set sha256 = "1c11726f1e8ace8bbdfc87816403c9a59f53a8c3d45c99187ae17c9725d87a91" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - six
+        - numpy
+        - pytools
+    run:
+        - python
+        - numpy
+        - pytools
+        - six
+
+test:
+    imports:
+        - {{ name }}
+
+
+about:
+    home: http://pypi.python.org/pypi/{{ name }}
+    license: MIT
+    license_file: LICENSE
+    summary: AST-based Python source generation
+
+    dev_url: http://github.com/inducer/{{ name }}
+
```

Comment:
I was going to suggest adding a `doc_url` for the documentation, but the docs link on the pypi page is dead. Just wanted to mention this in case you didn't realize it already.